cli/commands/bridge/to-l2.ts

```diff
@@ -161,7 +163,7 @@ export const redeemSendToL2 = async (cli: CLIEnvironment, cliArgs: CLIArgs): Pro
 }
 
 export const sendToL2Command = {
-  command: 'send-to-l2 <amount> [recipient]',
+  command: 'send-to-l2 <amount> [recipient] [calldata]',
   describe: 'Perform an L1-to-L2 Graph Token transaction',
   builder: (yargs: Argv): Argv => {
     return yargs
```

Comment:
Oh, actually, I think you might need to add the calldata option below here as well? i.e.

```
.positional('calldata', { demandOption: false })
```

Reply:
It's only needed if you want to describe it or pass any special argument. I'll do it and add descriptions.